### PR TITLE
feat(RichTextEditor): add column break button

### PR DESCRIPTION
**Pull request by celine-s**

### CODE REVIEW

File: src/components/RichTextEditor/Plugins/index.ts [hunk 1]
```diff
@@ -44,6 +45,32 @@ defaultPlugins
         ],
     );
 
+export const columnBreakPlugins = new PluginComposer();
```
Comment (mike-schmid) [inline src/components/RichTextEditor/Plugins/index.ts:48]: would it make sense to rename this const to something like "defaultPluginsWithColumns"? with the current name it sounds as if there are only column plugins

File: src/components/RichTextEditor/RichTextEditor.cy.tsx [hunk 2]
```diff
@@ -47,6 +48,28 @@ const LINK_CHOOSER_CHECKBOX = '.tw-group > .tw-inline-flex > .tw-flex-1 > .tw-se
 
 const insertTextAndOpenToolbar = () => cy.get('[contenteditable=true]').click().type('hello{selectall}');
 
+function checkPosition(chainers: string, value: number, text: string) {
```
Comment (mike-schmid) [inline src/components/RichTextEditor/RichTextEditor.cy.tsx:51]: All other functions in this file are declared as const arrow functions. We should keep it consistent and do it the same way here.
`const checkPosition = (chainers: string, value: number, text: string) => { ...`

File: src/components/RichTextEditor/RichTextEditor.cy.tsx [hunk 3]
```diff
@@ -47,6 +48,28 @@ const LINK_CHOOSER_CHECKBOX = '.tw-group > .tw-inline-flex > .tw-flex-1 > .tw-se
...
+        cy.get('[contenteditable=true]')
+            .contains(text)
+            .then((el) => {
+                const position = { left: el.position().left };
+                cy.wrap(position).its('left').should(chainers, value);
+            });
+    });
+}
+
+function selectTextValue(value: string) {
```
Comment (mike-schmid) [inline src/components/RichTextEditor/RichTextEditor.cy.tsx:62]: same as above :)

File: src/components/RichTextEditor/Plugins/ColumnBreakPlugin/ColumnBreakButton/ColumnBreakToolbarButton.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+export const ColumnBreakToolbarButton = ({
+    id,
+    pluginKey = ELEMENT_COLUMN_BREAK,
+    ...props
+}: ColumnBreakToolbarButtonProps) => {
+    const editor = usePlateEditorState(useEventPlateId(id));
+    const active = !!editor?.selection && someNode(editor, { match: { columnBreak: true } });
+    const [isActive, setIsActive] = useState(active ? true : false);
```
[the remark this answers is not in the document]
Comment (celine-s) [reply src/components/RichTextEditor/Plugins/ColumnBreakPlugin/ColumnBreakButton/ColumnBreakToolbarButton.tsx:26]: true, thanks!

File: src/components/RichTextEditor/Plugins/ColumnBreakPlugin/ColumnBreakButton/ColumnBreakToolbarButton.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    return (
+        <ToolbarButton
+            active={isActive}
+            onMouseDown={(event) => {
+                const setActive = getPreventDefaultHandler(setColumnBreak, editor, {
+                    value: isActive ? false : true,
+                    key: pluginKey,
+                });
+                setActive(event);
+                setIsActive(isActive ? false : true);
```
[the remark this answers is not in the document]
Comment (celine-s) [reply src/components/RichTextEditor/Plugins/ColumnBreakPlugin/ColumnBreakButton/ColumnBreakToolbarButton.tsx:37]: 👍

File: src/components/RichTextEditor/Plugins/ColumnBreakPlugin/ColumnBreakButton/index.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { IconSize, IconTextColumnBreak } from '@foundation/Icon';
+import { ButtonWrapper, IconStylingWrapper, buttonClassNames, buttonStyles } from '../../helper';
+import { PluginButtonProps } from '../../types';
+import { ColumnBreakToolbarButton } from './ColumnBreakToolbarButton';
+
+export const ColumnBreakButton = ({ id }: PluginButtonProps) => {
+    return (
```
[the remark this answers is not in the document]
Comment (celine-s) [reply src/components/RichTextEditor/Plugins/ColumnBreakPlugin/ColumnBreakButton/index.tsx:10]: 👍

File: src/components/RichTextEditor/RichTextEditor.cy.tsx (hunk 2 above)
Comment (celine-s) [reply src/components/RichTextEditor/RichTextEditor.cy.tsx:51]: true, thanks!

File: src/components/RichTextEditor/Plugins/ColumnBreakPlugin/ColumnBreakButton/ColumnBreakToolbarButton.tsx [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+    return (
+        <ToolbarButton
+            active={isActive}
+            onMouseDown={(event) => {
+                const setActive = getPreventDefaultHandler(setColumnBreak, editor, {
+                    value: isActive ? false : true,
+                    key: pluginKey,
+                });
+                setActive(event);
```
[the remark this answers is not in the document]
Comment (celine-s) [reply src/components/RichTextEditor/Plugins/ColumnBreakPlugin/ColumnBreakButton/ColumnBreakToolbarButton.tsx:36]: 👍

File: src/components/RichTextEditor/Plugins/ColumnBreakPlugin/setColumnBreak.ts [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+        unsetNodes(editor, nodeKey!, {
+            match,
+        });
+    } else {
+        setElements(
+            editor,
+            { [nodeKey!]: value },
+            {
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                match: match as any,
```
[the remark this answers is not in the document]
Comment (celine-s) [reply src/components/RichTextEditor/Plugins/ColumnBreakPlugin/setColumnBreak.ts:37]: removed 👍

**Review by mike-schmid — APPROVED** [review]

**Pull request merged**